integrations/airflow/marquez_airflow/dag.py

```diff
@@ -41,9 +40,44 @@
 _MARQUEZ = MarquezAdapter()
 
 
+@provide_session
+def task_run_id(run_id, task, session=None):
```

Comment:
Given that Airflow also has a notion of task runs, we might want to be explicate here and name the function `marquez_task_run_id` or similar.

Reply:
I did have a similar thought. I left it as is just for brevity, but it's probably better to be more explicit.